frontend/packages/ux-editor/src/components/AddOption.tsx

```diff
@@ -97,6 +102,7 @@ export const AddOption = <T extends FormCheckboxesComponent | FormRadioButtonsCo
       >
         <span className={addButtonClass}>
           <PlusIcon />
+          <span>jgfgfgfgfgfgfgfgfgfgfgfgfgfgfgfgfgfgfgfgfgfgfgfgfgf</span>
```

Comment:
hehe, some debug texts that you forgot to remove before committing?

Reply:
oh my God :see_no_evil: I will make my own checkbox-list before any commit